wallet/res/navigation/nav_home.xml

```diff
@@ -324,6 +337,18 @@
             android:defaultValue="false" />
     </fragment>
 
+    <fragment
+        android:id="@+id/showNotificationsFragment"
+        android:name="de.schildbach.wallet.ui.dashpay.NotificationsFragment"
+        android:label="ShowSeed"
+        tools:layout="@layout/fragment_notifications">
+
+        <argument
+            android:name="extra"
```

Comment:
Is this the correct argument name? "mode" is used below:

`bundleOf("mode" to NotificationsFragment.MODE_NOTIFICATIONS),`